bindings/node/lib/stateMachine.js

```diff
@@ -280,7 +281,7 @@ module.exports = function(modules) {
                 command: 'connect',
                 destination: { host: options.host, port: options.port },
                 proxy: {
-                  host: 'locahost',
+                  host: 'localhost',
```

Comment:
what specific bug was this causing? can we test for it?

Reply:
There is no bug! My mistake, I forgot the way the proxy API works is that the existing_socket is already connected above but you still have to pass in these options, I've udpated the code to match what we put in the driver :)